sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClient.java

```diff
@@ -1511,35 +1511,33 @@ public Response<BinaryData> getImageGenerationsWithResponse(String deploymentOrM
      * @return the result of a successful image generation operation along with {@link Response}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<ImageGenerations> getImageGenerationsWithResponse(String deploymentOrModelName,
+    Response<ImageGenerations> getImageGenerationsWithResponse(String deploymentOrModelName,
         ImageGenerationOptions imageGenerationOptions, RequestOptions requestOptions) {
         Response<BinaryData> response = getImageGenerationsWithResponse(deploymentOrModelName,
             BinaryData.fromObject(imageGenerationOptions), requestOptions);
         return new SimpleResponse<>(response, response.getValue().toObject(ImageGenerations.class));
     }
 
-    /**
-     * Creates an image given a prompt.
-     *
-     * @param deploymentOrModelName Specifies either the model deployment name (when using Azure OpenAI) or model name
-     * (when using non-Azure OpenAI) to use for this request.
-     * @param imageGenerationOptions Represents the request data used to generate images.
-     * @throws IllegalArgumentException thrown if parameters fail the validation.
-     * @throws HttpResponseException thrown if the request is rejected by server.
-     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
-     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
-     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the result of a successful image generation operation.
-     */
```

Comment:
Somehow codegen now gets rid of comments. Do you know if this can be somehow changed @mssfang ?

Reply:
Maybe we can try to delete `// @Generated` and manually added those back.